scripts/test-runner.sh

```diff
@@ -13,9 +13,9 @@ BACKEND=${ORIGARGS[0]}
 MODE=${ORIGARGS[1]}
 ENGINE=${ORIGARGS[2]}
 
-SUPPORTED_BACKENDS=("geth" "ganache")
-SUPPORTED_MODE=("http" "ws")
-SUPPORTED_ENGINES=("node" "electron" "firefox" "chrome" "")
+SUPPORTED_BACKENDS=("geth" "ganache" "ipc")
+SUPPORTED_MODE=("http" "ws" "ipc")
+SUPPORTED_ENGINES=("node" "electron" "firefox" "chrome" "ipc" "")
```

Comment:
We should run the IPC tests with `node` or `electron` 

```suggestion
SUPPORTED_ENGINES=("node" "electron" "firefox" "chrome" "")
```

Reply:
IpcProvider in the electron from cypress can't connect to the IPC file. We've, with @jdevcs, decided to skip an electron to the IPC test